### PR TITLE
Supports dropping multiple tables at the same time

### PR DESCRIPTION
**Pull request by bakam412**
Original issue ： #50572 

### Changelog category (leave one):
- New Feature

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Supports dropping multiple tables at the same time like `drop table a,b,c;`

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

### CODE REVIEW

**Comment by bakam412** [comment]
I apologize for forgetting to associate this pull request with the related issue. Therefore, here is the additional information regarding the associated issue: https://github.com/ClickHouse/ClickHouse/issues/50572

**Comment by robot-clickhouse-ci-2** [comment]
This is an automated comment for commit 47ac18a94032522e2d52b2870d9ab6c95637fc92 with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/52191/47ac18a94032522e2d52b2870d9ab6c95637fc92/ci_running.html)
The overall status of the commit is 🔴 failure

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td>🔴 failure</td></tr>
<tr><td>Docs Check</td><td>Builds and tests the documentation</td><td>🟡 pending</td></tr>
</table>

**Comment by hanfei1991** [comment]
Could you please update the docs as well?

**Comment by bakam412** [comment]
> Could you please update the docs as well?

updated

File: src/Parsers/ASTQueryWithTableAndOutput.h [hunk 1]
```diff
@@ -16,6 +16,7 @@ class ASTQueryWithTableAndOutput : public ASTQueryWithOutput
 public:
     ASTPtr database;
     ASTPtr table;
```
[the remark this answers is not in the document]
Comment (bakam412) [reply src/Parsers/ASTQueryWithTableAndOutput.h:18]: Those variables are not used for the Drop query in this feature, but another subclass uses them. So we should not delete them from this class.

[the remark this answers is not in the document]
Comment (bakam412) [reply src/Parsers/ASTQueryWithTableAndOutput.h:18]: Or I can declare it in class ASTDropQuery

**Comment by hanfei1991** [comment]
fast test meets lots of timeout when dropping databases https://s3.amazonaws.com/clickhouse-test-reports/52191/7425ff0475af5b16e1b146e7f84c15a9be0e6650/fast_test/run.log

**Comment by bakam412** [comment]
> fast test meets lots of timeout when dropping databases https://s3.amazonaws.com/clickhouse-test-reports/52191/7425ff0475af5b16e1b146e7f84c15a9be0e6650/fast_test/run.log

Ok， I will check it

**Comment by alexey-milovidov** [comment]
@bakam412, how is it?

**Comment by bakam412** [comment]
> @bakam412, how is it?

This feature conflicts with someone else's code, and I am currently debugging it while integrating it with their code

**Comment by alexey-milovidov** [comment]
@bakam412 does it have a chance?

**Comment by alexey-milovidov** [comment]
Closing due to no answer.

**Pull request closed without merging**